app/views/recipes/index.html.erb

```diff
@@ -0,0 +1,6 @@
+<% @recipes.each do |recipe| %>
+  <%= render 'shared/recipe', recipe: recipe %>
+<%end%>
```

Comment:
- [ ] As requested by the previous reviewer, please make sure the style (layout) list of your recipes page (recipe index page) matches the given wireframe. :pray: 
  
  ### Your design
  
  ![image](https://github.com/ezraermy/Recipe-App/assets/92303891/c893dacf-aa25-4539-a878-68e08c34799f)
  
  ### Given wireframe
  
  ![image](https://github.com/ezraermy/Recipe-App/assets/92303891/7e75ed5d-786b-4fa0-8f4e-123b455f3f04)